static/app/gettingStartedDocs/elixir/elixir.tsx

```diff
@@ -138,8 +138,8 @@ environment_name: System.get_env("RELEASE_LEVEL") || "development"
             </p>
             <p>
               {tct(
-                'If using an environment with Plug or Phoenix, add the following to [codePlugRouter:Plug.Router] or [codePhoenixEndpoint:Phoenix.Endpoint]:',
-                {codePlugRouter: <code />, codePhoenixEndpoint: <code />}
```

Comment:
These don't need renamed.

Reply:
good catch! but actually we need just to remove the prefix :) renamed to be consistent with other files